### DISCUSSION TITLE
[New] Proposition d'ajout de pratique : « Choisir des valeurs par défaut favorisant un impact reduit »

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#26] [Bonnes pratiques] [answered]
Les utilisateurs laissent souvent les valeurs par défaut. Cela peut être utilisé à bon escient.

L'exemple le plus flagrant étant la résolution vidéo : on peut avoir par défaut une résolution acceptable (480p par ex), et laisser ceux qui le souhaite choisir une résolution plus grande.

**Comment by ACTLEM** [comment]
A part la résolution vidéo, y a-t-il d'autres exemples? 

Si non, n'est-ce pas une amélioration de la BP 114 "Adapter les vidéos aux contextes
de visualisation" voire indirectement lié à a BP 6 "Préférer l’approche « mobile first" ?

**Reply by rlemaire** [reply]
Ca peut être aussi pour des exports de données (plus ou moins de lignes et de colonnes), des paramètres de recherche (limitation du nombre de résultat, ...), tout ce qui potentiellement peut déplacer pas mal de données.

**Reply by ACTLEM** [reply]
Ok, je comprends mieux l'idée. Sur les paramètres de recherche cela semble rejoindre la BP "Quantifier précisément le besoin".
Je me demande s'il ne faudrait pas plutôt une BP pour chaque cas, ou avoir cette BP mais avec un maximum d'exemples. Sur "l'impact réduit", si l'idée est de réduire les données, je remplacerait donc par "...favorisant une réduction des données échangées".

**Reply by ACTLEM** [reply]
Ensuite, il ne me semble pas évident de trouver une règle de validation simple à appliquer si cela reste trop générique.

**Reply by rlemaire** [reply]
Ça me parait différent de "Quantifier précisément le besoin" qui est plus au niveau architecture, choix techniques à la réalisation alors qu'ici on est sur l'interface, l'UX, et que l’utilisateur reste libre de changer la valeur.

Effectivement on est ici dans un cas où il n'y aura pas de règles de validations, comme pour "Fluidifier le processus" ou "Éliminer les fonctionnalités non essentielles".

**Reply by ACTLEM** [reply]
Effectivement. La description devra mentionner cet aspect interface utilisateur. Pour le titre, j'ajouterais la notion d'utilisateur, par exemple (mais sûrement améliorable): "Définir pour l'utilisateur des valeur par défaut favorisant un impact réduit"

**Reply by ACTLEM** [reply]
En complément, plutôt qu'impact réduit, je mettrais: "Définir pour l'utilisateur des valeur par défaut au minimum suffisant" dans le sens minimum suffisant pour une expérience, un service rendu sans dégradation.

**Comment by Trankilloman** [comment]
Personnellement j'ai du mal avec le "impact réduit" car l'utilisateur du guide n'a pas forcément le recul ni les outils pour l'estimer.

**Reply by rlemaire** [reply]
Ca me parait raisonnable de supposer que l'utilisateur a les grands principes en tête lors de la lecture. Sinon comment même peut-il choisir quelle pratique appliquer ou non, avec quelle priorité, pourquoi ?

**Comment by ACTLEM** [comment, marked as answer]
Note équipe projet: Amélioration de la BP 2.

Issue créée: https://github.com/cnumr/best-practices/issues/184
